### DISCUSSION TITLE
Jodit Editor - Plan Inquiry

### DISCUSSION THREAD

**Discussion by Ramnaresh9398** [xdan/jodit#1047] [Q&A] [answered]
Hello,

We intend to use Jodit editor on a website built on MERN (MongoDB, Express, React, Node) and host this as a SAAS platform for end users outside our organization. This platform may exist across multiple domains. However, we do not plan to integrate jodit with any open source products. In this scenario, will professional plan be sufficient ?

**Comment by xdan** [comment]
> multiple domains
The Multi Pro license is right for you

**Reply by Ramnaresh9398** [reply]
Thank you for your response! It means a lot.

**Comment by xdan** [comment, marked as answer]
Multi Pro is the right fit. The distinction that matters: OEM/redistribution is only needed if you bundle the editor into a product that other people install and run themselves. In your case the editor runs on your own (multiple) domains and your users just use your SaaS — that's not redistribution, so unlimited-domain Multi Pro covers it. Professional is single-domain, so it wouldn't be enough across multiple domains.
